### DISCUSSION TITLE
What does the red part at the top of the layers mean?

### DISCUSSION THREAD

**Discussion by rfresh737** [sn4k3/UVtools#480] [Q&A] [answered]
![2022-05-19_184520](https://user-images.githubusercontent.com/10987314/169593981-a13ff4dd-692d-4bba-8a28-bcfcb6e03f36.png)

**Comment by sn4k3** [comment, marked as answer]
Red can be empty layer or touching bounds. My guess is empty layer as it's the last layer.
Colors can be seen and configured on Settings - Layer preview

**Comment by rfresh737** [comment]
It must be an empty layer in this case because that layer does not print at all.
Thanks...
